config/settings.yml

```diff
@@ -116,6 +116,7 @@ features:
     state: confirmed # final allocation places are displayed to users in a readonly state
   visa_sponsorship_on_course: true
   engineers_teach_physics_on_course: true
+  user_self_management: false
```

Comment:
```suggestion
user_management: false
```

Reply:
Hmm, I was going to call the feature flag this but I thought it sounded like we are managing the users, not the users self serving. 

Anyways im not that fussed, we'll be deleting this feature flag sooon. Happy to change it to `user_management` if you still prefer that?

Reply:
Yeah, I don't love the other one 😂

Reply:
😅 🪦 

https://github.com/DFE-Digital/publish-teacher-training/pull/3154/commits/8e501a0a28af6ba6f91c8468fdbbf6e69635d00c